src/main/java/org/apache/commons/lang3/math/Fraction.java

```diff
@@ -31,8 +31,11 @@
  * based and thus suffers from various overflow issues. For a BigInteger based
  * equivalent, please see the Commons Math BigFraction class. </p>
  *
+ * @deprecated for <a href="https://commons.apache.org/proper/commons-math/javadocs/api-3.6.1/org/apache/commons/math3/fraction/Fraction.html">Fraction</a>
```

Comment:
We can't just deprecate the class without documenting how to port your current code to the other API IMO. For example, what do you do with `Fraction#invert()`? Each method must be documented, otherwise, it's a disservice, and everyone using the class is left having to do their own investigation and mapping.